flink-clients/src/main/java/org/apache/flink/client/cli/ProgramOptions.java

```diff
@@ -56,8 +55,6 @@
 
 	private final boolean stdoutLogging;
 
-	private final boolean detachedMode;
```

Comment:
I think we should not remove these fields. The `ProgramOptions` represent general options for running a program. The execution behaviour is in my opinion one of these options.